import-export-cli/cmd/getApis.go

```diff
@@ -65,6 +66,7 @@ var getApisCmd = &cobra.Command{
 
 func executeGetApisCmd(credential credentials.Credential) {
 	accessToken, err := credentials.GetOAuthAccessToken(credential, getApisCmdEnvironment)
+	fmt.Println("Access Token: ", accessToken)
```

Comment:
Do we need this?

Reply:
Fixed by [02a788a](https://github.com/wso2/product-apim-tooling/pull/924/commits/02a788a1570035b84d7acfea4bc999161a1bae47)